internal/pkg/local/delete_test.go

```diff
@@ -14,7 +14,7 @@ func TestLocalDeleteModel(t *testing.T) {
 	fs := manager.fs
 
 	record := &MockedRecord{}
-	assert.NoError(t, manager.manifest.TrackRecord(record))
```

Comment:
V testoch sa volalo niekde `PersistRecord` a niekde `TrackRecord`, kedze v danych testoch to bolo jedno.
`trackRecord` je uz private, pouziva sa iba interne, ... takze som to upravil v testoch na `PersistRecord`.

Takto je interface jednoduchsi a neda sa uz pomylit, co sa ma volat.